core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java

```diff
@@ -1,78 +1,335 @@
+import software.amazon.awssdk.services.applicationautoscaling.model.ServiceNamespace;
+import software.amazon.awssdk.services.applicationautoscaling.model.TargetTrackingScalingPolicyConfiguration;
 import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
 import software.amazon.awssdk.services.dynamodb.DynamoDbClientBuilder;
+import software.amazon.awssdk.services.dynamodb.model.AttributeDefinition;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+import software.amazon.awssdk.services.dynamodb.model.CreateTableRequest;
+import software.amazon.awssdk.services.dynamodb.model.DeleteItemRequest;
+import software.amazon.awssdk.services.dynamodb.model.DeleteTableRequest;
+import software.amazon.awssdk.services.dynamodb.model.DescribeTableRequest;
+import software.amazon.awssdk.services.dynamodb.model.DescribeTableResponse;
+import software.amazon.awssdk.services.dynamodb.model.DynamoDbException;
+import software.amazon.awssdk.services.dynamodb.model.GlobalSecondaryIndex;
+import software.amazon.awssdk.services.dynamodb.model.KeySchemaElement;
+import software.amazon.awssdk.services.dynamodb.model.KeyType;
+import software.amazon.awssdk.services.dynamodb.model.ListTablesResponse;
+import software.amazon.awssdk.services.dynamodb.model.LocalSecondaryIndex;
+import software.amazon.awssdk.services.dynamodb.model.PointInTimeRecoverySpecification;
+import software.amazon.awssdk.services.dynamodb.model.Projection;
+import software.amazon.awssdk.services.dynamodb.model.ProjectionType;
+import software.amazon.awssdk.services.dynamodb.model.ProvisionedThroughput;
+import software.amazon.awssdk.services.dynamodb.model.ResourceNotFoundException;
+import software.amazon.awssdk.services.dynamodb.model.ScalarAttributeType;
+import software.amazon.awssdk.services.dynamodb.model.ScanRequest;
+import software.amazon.awssdk.services.dynamodb.model.ScanResponse;
+import software.amazon.awssdk.services.dynamodb.model.TableStatus;
+import software.amazon.awssdk.services.dynamodb.model.UpdateContinuousBackupsRequest;
 
 @ThreadSafe
 public class DynamoAdmin implements DistributedStorageAdmin {
+  private static final Logger LOGGER = LoggerFactory.getLogger(DynamoAdmin.class);
+  private static final String PARTITION_KEY = "concatenatedPartitionKey";
+  private static final String CLUSTERING_KEY = "concatenatedClusteringKey";
+  private static final String GLOBAL_INDEX_NAME_PREFIX = "global_index";
+  private static final int CREATING_WAITING_TIME = 3000;
+  private static final int COOL_TIME_SEC = 60;
+  private static final double TARGET_USAGE_RATE = 70.0;
+  private static final int DELETE_BATCH_SIZE = 100;
+
+  private static final String NO_SCALING = "no-scaling";
+  private static final String NO_BACKUP = "no-backup";
+  private static final String REQUEST_UNIT = "ru";
+  private static final Boolean DEFAULT_NO_SCALING = false;
+  private static final Boolean DEFAULT_NO_BACKUP = false;
+  private static final long DEFAULT_RU = 10;
```

Comment:
Please make those `public`:
```suggestion
  public static final String NO_SCALING = "no-scaling";
  public static final String NO_BACKUP = "no-backup";
  public static final String REQUEST_UNIT = "ru";
  public static final Boolean DEFAULT_NO_SCALING = false;
  public static final Boolean DEFAULT_NO_BACKUP = false;
  public static final long DEFAULT_RU = 10;
```